app/javascript/components/admin/AdminTable.js

```diff
@@ -475,6 +515,7 @@ class AdminTable extends React.Component {
         const csvData = response.data.user_rows.map(userData => {
           return {
             ...userData,
+            notes: this.removeFormulaStart(userData.notes).toString(),
```

Comment:
should we just start preventing users from being able to start notes with formula values instead of allowing them to and then deleting it without them knowing? @holmesie

Reply:
seeing as @sgober and @jkufro seem to agree with this, can we implement that instead?

Reply:
A new ticket will be created by @holmesie for this issue.